Buildings/Fluid/CHPs/OrganicRankine/Cycle.mo

```diff
@@ -0,0 +1,493 @@
+
+  replaceable parameter Buildings.Fluid.CHPs.OrganicRankine.Data.Generic pro
+    constrainedby Buildings.Fluid.CHPs.OrganicRankine.Data.Generic
+    "Property records of the working fluid"
+    annotation(choicesAllMatching = true,Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.MassFlowRate mHot_flow_nominal
+    "Nominal mass flow rate of the evaporator fluid"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.PressureDifference dpHot_nominal = 0
+    "Nominal pressure drop of the hot fluid in evaporator"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.TemperatureDifference dTPinEva_set(
+    final min = 0) = 5
+    "Set evaporator pinch point temperature difference"
+    annotation(Dialog(group="Evaporator"));
+  parameter Boolean useEvaporatingPressure = false
+    "Set true to specify working fluid evaporating pressure instead of temperature"
+    annotation(Dialog(group="Evaporator"));
+  parameter Modelica.Units.SI.ThermodynamicTemperature TWorEva =
+    max(pro.T)*2/3 + min(pro.T)*1/3
+    "Evaporating temperature of the working fluid"
+    annotation(Dialog(group="Evaporator", enable = not useEvaporatingPressure));
+  parameter Modelica.Units.SI.Pressure pWorEva(displayUnit="kPa") =
+    max(pro.p)*2/3 + min(pro.p)*1/3
+    "Evaporating pressure of the working fluid"
+    annotation(Dialog(group="Evaporator", enable = useEvaporatingPressure));
+  parameter Modelica.Units.SI.MassFlowRate mCol_flow_nominal
+    "Nominal mass flow rate of the condenser fluid"
+    annotation(Dialog(group="Condenser"));
+  parameter Modelica.Units.SI.PressureDifference dpCol_nominal = 0
+    "Nominal pressure drop of the cold fluid in condenser"
+    annotation(Dialog(group="Condenser"));
+  parameter Modelica.Units.SI.TemperatureDifference dTPinCon(
+    final min = 0) = 10
+    "Condenser pinch point temperature difference"
+    annotation(Dialog(group="Condenser"));
+  parameter Boolean useLowCondenserPressureWarning = true
+    "If true, issues warning if pCon < 101325 Pa"
+    annotation(Dialog(group="Condenser"));
+  parameter Modelica.Units.SI.MassFlowRate mWor_flow_max(
+    final min = 0)
+    "Upper bound of working fluid flow rate"
+    annotation(Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.MassFlowRate mWor_flow_min(
+    final min = 0)
+    "Lower bound of working fluid flow rate"
+    annotation(Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.MassFlowRate mWor_flow_hysteresis
+    = mWor_flow_min + (mWor_flow_max - mWor_flow_min) * 0.1
+    "Hysteresis for turning off the cycle when flow too low"
+    annotation(Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.Efficiency etaExp
+    "Expander efficiency"
+    annotation(Dialog(group="Cycle"));
+  parameter Modelica.Units.SI.Efficiency etaPum
+    "Pump efficiency"
+    annotation(Dialog(group="Cycle"));
+
+  Modelica.Blocks.Interfaces.RealOutput PEle(
```

Comment:
Move all input and output connectors to the outside of the icon, such as for example in `Carnot_y`, and also move the evaporator and condenser to the right rather than top and bottom.

Reply:
Addressed in 257a1f374c05fbf571cb3eb1c6526fd037ba6a30.